From ec454967c723eec599a03b6747eb5f4209360f31 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Mon, 30 Dec 2019 08:32:30 +0100
Subject: [PATCH] Switch erml2pdf.py to Python 3

Even if kraft runs it with an explicit Python interpreter, switch its
shebang to Python 3 to make sure it will be also when run as standalone
executable.
---
 tools/erml2pdf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/erml2pdf.py b/tools/erml2pdf.py
index f5f4ea0..9f35c2d 100644
--- a/tools/erml2pdf.py
+++ b/tools/erml2pdf.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 #
 # erml2pdf - An RML to PDF converter with extended features
